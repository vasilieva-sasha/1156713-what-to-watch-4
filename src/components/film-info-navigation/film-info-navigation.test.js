import React from "react";
import renderer from "react-test-renderer";
import FilmInfoNavigation from "./film-info-navigation";

describe(`FilmNavigationComponent`, () => {
  it(`Navigation bar renders correctly`, () => {
    const tree = renderer.create(
        <FilmInfoNavigation onTabClick={() => {}} />
    );

    expect(tree).toMatchSnapshot();
  });
});
