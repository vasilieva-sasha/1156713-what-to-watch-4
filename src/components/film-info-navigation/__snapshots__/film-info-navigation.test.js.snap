// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FilmNavigationComponent Navigation bar renders correctly 1`] = `
<nav
  className="movie-nav movie-card__nav"
>
  <ul
    className="movie-nav__list"
  >
    <li
      className="movie-nav__item movie-nav__item--active"
    >
      <a
        className="movie-nav__link"
        href="#"
        onClick={[Function]}
      >
        Overview
      </a>
    </li>
    <li
      className="movie-nav__item "
    >
      <a
        className="movie-nav__link"
        href="#"
        onClick={[Function]}
      >
        Details
      </a>
    </li>
    <li
      className="movie-nav__item "
    >
      <a
        className="movie-nav__link"
        href="#"
        onClick={[Function]}
      >
        Reviews
      </a>
    </li>
  </ul>
</nav>
`;
